doc/src/manual/noteworthy-differences.md

```diff
@@ -209,7 +209,7 @@ For users coming to Julia from R, these are some noteworthy differences:
   * Python Lists—flexible but slow—correspond to the Julia `Vector{Any}` type or more generally `Vector{T}` where `T` is some non-concrete element type. "Fast" arrays like NumPy arrays that store elements in-place (i.e., `dtype` is `np.float64`, `[('f1', np.uint64), ('f2', np.int32)]`, etc.) can be represented by `Array{T}` where `T` is a concrete, immutable element type. This includes built-in types like `Float64`, `Int32`, `Int64` but also more complex types like `Tuple{UInt64,Float64}` and many user-defined types as well.
   * In Julia, indexing of arrays, strings, etc. is 1-based not 0-based.
   * Julia's slice indexing includes the last element, unlike in Python. `a[2:3]` in Julia is `a[1:3]`
-    in Python.
+    in Python (note that the the start and stop values are also shifted by one here because of the differences in 0-based and 1-based indexing in the two languages).
```

Comment:
Maybe this can be show-not-tell, something like this?
```suggestion
  * Julia's slice indexing includes the last element, unlike in Python. Thus `ju[2:3]` in Julia is `py[1:3]`
    in Python -- both omit the first element (`ju[1]` or `py[0]`) and then keep the next two.
```

Reply:
I think it's important for the first time reader to be more explicit that the radix of the languages is factoring into both start and stop values. Perhaps the 2nd commit I just added addresses your concern?

Reply:
I'm OK with saying "first" for the first, partly as a jab at zero-based indexing, but saying "2nd" seems unclear. 

If you think the previous bullet point should say more about what zero-based means, perhaps expand that. But I wouldn't try to squeeze a whole explanation of that into a paranthetical note about closed-end ranges.

Reply:
Another way to clarify this might be to make the example more explicit. `collect("abcd")[2:3]` in Julia and `list('abcd')[1:3]` in Python both contain letters 'b' and 'c' only.